libraries/tock-register-interface/README.md

```diff
@@ -262,6 +260,17 @@ ReadWrite<T: IntLike, R: RegisterLongName = ()>
 .matches_all(value: FieldValue<T, R>) -> bool  // Check if all specified parts of a field match
 .extract() -> LocalRegisterCopy<T, R>          // Make local copy of register
 
+Aliased<T: IntLike, R: RegisterLongName = (), W: RegisterLongName = ()>
+.get() -> T                                    // Get the raw register value
+.set(value: T)                                 // Set the raw register value
+.read(field: Field<T, R>) -> T                 // Read the value of the given field
```

Comment:
I think the documentation would benefit from some explicit explanation of what an aliased register is, and then here making it explicit/clear that the read and write methods operate on different things

Reply:
I added a much more expressive comment in the rustdoc for the `Aliased` struct.  IMO, it would be good to strike a balance between that and the README.  Syncing and duplicating between them would be a growing hassle.  (And the `external_doc` feature which would help that is still marked unstable)